integration/spark2/src/main/scala/org/apache/spark/sql/CarbonSession.scala

```diff
@@ -43,7 +43,7 @@ class CarbonSession(@transient val sc: SparkContext,
   }
 
   @transient
-  override private[sql] lazy val sessionState: SessionState = new CarbonSessionState(this)
+  override lazy val sessionState: SessionState = new CarbonSessionState(this)
```

Comment:
Is this change required? Somewhere outside sql package will read this variable?

Reply:
Yes, it is required for test case I added.